installer/main.tf

```diff
@@ -17,6 +17,12 @@
 data "google_compute_default_service_account" "default" {
 }
 
+resource "random_string" "backend_name" {
```

Comment:
The idea here is to allow for multiple Tightlock VMs on the same project?

Reply:
Yes, this would allow multiple instances to be added without a naming clash. Also added `Tightlock` to the VPC name.